setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Would be good to set options`-eu` to enable a stricter mode that ensures undefined variable use and failed commands cause execution to stop instead of continue.